test_engine/config.yml

```diff
@@ -0,0 +1,4 @@
+"nodes":
```

Comment:
@cchien816 can you elaborate what config.yml is used for?
if the file is automatically generated, we don't need to commit it to the repo.

Reply:
Yes, it's auto generated, I will remove from next commit